packages/catalog-client/api-report.md

```diff
@@ -178,6 +186,26 @@ export interface GetEntityAncestorsResponse {
   rootEntityRef: string;
 }
 
+// @public
+export interface GetEntityFacetsRequest {
+  facets: string[];
+  filter?:
+    | Record<string, string | symbol | (string | symbol)[]>[]
+    | Record<string, string | symbol | (string | symbol)[]>
+    | undefined;
```

Comment:
I'm feeling this API is a little bit unfortunate to use all the way out in plugins and components tbh. As an API layer it's probably fine, but I'm kinda missing the possibility to communicate things like what facets one should care about for example. What are your thoughts on an API that sits in-between this and plugins? Perhaps making things like `count` optional

Reply:
> Not sure how to parse "communicate things like what facets one should care about" honestly 🤔

I'm thinking things like hand-crafting the available values if you're doing something like a dropdown to select a lifecycle, or to be able to statically declare which entity kinds have lifecycles and which don't, regardless of what the data in the catalog looks like.

Overall just feels a icky to have this tight coupling all the way from the frontend components to the backend, but tbh it's probably fine too because I don't think we'd want to spend time designing any intermediate API now anyway.

> What's to be gained by making count optional? Do you mean making it possible to make faster lookups by skipping counts?

That or client-side only implementations if one wants to go down that route.

Reply:
Alright, yeah I see where you are coming from and a nicer intermediate layer could make sense eventually to make this less harsh and not so backend-1:1-shaped. But indeed, probably not right now

Reply:
Marking as resolved for now, feel free to reopen of course if that was too hasty